src/main/java/org/pac4j/demo/dw/Pac4JDemoApplication.java

```diff
@@ -45,6 +47,12 @@ protected void configure() {
         });
 
         env.jersey().register(ViewsResource.class);
-    }
 
+        env.healthChecks().register(getName(), new HealthCheck() {
+            @Override
+            protected Result check() throws Exception {
+                return Result.healthy();
```

Comment:
Ok, I will make an issue in dropwizard-pac4j so that we don't forget it. Basically dw uses healthchecks as a way to quickly check all external dependency of an application, such as database, and in this instance, connections to external identity service.
For example if the facebook token was incorrect (or expired say), an healthcheck could be there so that the administrator of the application can be notified of it.